<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrixConfigW</class>
 <widget class="QMainWindow" name="PrixConfigW">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changement des Prix</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
/*	background-image:url(image/Zen-Background-23.jpg);*/
	/*background-repeat:none;*/

/*	background-color:#97d562;
	background: qlineargradient(stop: 0 rgba(250,250,250,1), stop: 1 rgba(151,213,98,1));*/
/*	background: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(250,250,250,255), stop: 1 rgba(151,213,98,255));*/

background-color: qlineargradient( x1: 100, y1:100, x2: 200, y2: 200,
														stop: 0 rgba(250,250,250,255), stop: 1 rgba(151,213,98,255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_ajout">
      <property name="styleSheet">
       <string notr="true">QLabel 
{
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}
</string>
      </property>
      <property name="text">
       <string>Ajout soins/catégorie</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_intitule_categorie">
        <property name="styleSheet">
         <string notr="true">QLabel 
{
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}
</string>
        </property>
        <property name="text">
         <string>Intitulé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_categorie"/>
      </item>
      <item>
       <widget class="QPushButton" name="Add_categorie">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
        </property>
        <property name="text">
         <string>Ajouter categorie</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_soin_intitule">
        <property name="styleSheet">
         <string notr="true">QLabel 
{
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}
</string>
        </property>
        <property name="text">
         <string>Intitulé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_add_soin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_choix_categorie">
        <property name="styleSheet">
         <string notr="true">QLabel 
{
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}
</string>
        </property>
        <property name="text">
         <string>Catégorie associé</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_categorie"/>
      </item>
      <item>
       <widget class="QLabel" name="label_prx_nouveau_soins">
        <property name="styleSheet">
         <string notr="true">QLabel 
{
     font: bold 12px;
     min-width: 5em;
     padding: 7px;
}
</string>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spinBox_prix">
        <property name="maximum">
         <double>999.990000000000009</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Add_soins">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}


</string>
        </property>
        <property name="text">
         <string>Ajouter soin</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_show_en_cours">
      <property name="styleSheet">
       <string notr="true">QCheckBox 
{
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}
</string>
      </property>
      <property name="text">
       <string>Afficher les soins désactivés</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">QScrollArea
{
background: rgba(240,240,240,255)
}
</string>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>778</width>
         <height>506</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_reset">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
        </property>
        <property name="text">
         <string>Réinitialiser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ok">
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
        </property>
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
